### PR TITLE
initial

### PR DESCRIPTION
**Pull request by emalfroy**

### CODE REVIEW

File: src/Grb.Building.Api/Infrastructure/Startup.cs [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+                        Server =
+                        {
+                            BaseUrl = baseUrlForExceptions
+                        },
+                        Swagger =
+                        {
+                            ApiInfo = (provider, description) => new OpenApiInfo
+                            {
+                                Version = description.ApiVersion.ToString(),
+                                Title = "Basisregisters Vlaanderen Building Registry GRB API",
```
Comment (emalfroy) [inline src/Grb.Building.Api/Infrastructure/Startup.cs:79]: Title should be Basisregisters GRB API?

File: src/Grb.Building.Api/appsettings.json [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  },
+
+  "AWS": {
+    "Region": "eu-west-1"
+  },
+
+  "DataDog": {
+    "Enabled": true,
+    "Debug": false,
+    "ServiceName": "building-registry-grb-upload-api"
```
Comment (emalfroy) [inline src/Grb.Building.Api/appsettings.json:28]: @ArneD Do we need to change this to basisregisters-grb-upload-api?

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -0,0 +1,557 @@
...
+    if: needs.release.outputs.version != 'none'
+    needs: [ release ]
+    name: Push images to Test
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        image: [
+          'api-grb',
+          'grb-processor-upload',
+          'grb-processor-job',
```
Comment (jvandaal) [inline .github/workflows/release.yml:256]: The comma at the end will maybe cause problems?

File: src/Grb.Building.Api/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+# create group & user
+RUN addgroup --gid 1000 --system app && adduser --uid 1000 -system app --gid 1000
+
+# create work dir and set permissions as WORKDIR sets permissions as root
+RUN mkdir /app && chown -R app:app /app
+WORKDIR /app
+
+LABEL maintainer "Digitaal Vlaanderen <digitaal.vlaanderen@vlaanderen.be>"
+LABEL registry="building-registry"
```
Comment (emalfroy) [inline src/Grb.Building.Api/Dockerfile:11]: change to basisregisters-grb

File: .github/workflows/release.yml [hunk 5]
```diff
@@ -0,0 +1,557 @@
...
+    if: needs.release.outputs.version != 'none'
+    needs: [ release ]
+    name: Push images to Staging
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        image: [
+          'api-grb',
+          'grb-processor-upload',
+          'grb-processor-job',
```
Comment (jvandaal) [inline .github/workflows/release.yml:307]: Remove comma

File: src/Grb.Building.Api/Uploads/UploadPreSignedUrlHandler.cs [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+                    new CreatePresignedPostRequest(
+                        _bucketOptions.BucketName,
+                        job.UploadBlobName,
+                        new List<ExactMatchCondition>(),
+                        TimeSpan.FromMinutes(_bucketOptions.UrlExpirationInMinutes)));
+
+                var ticketId= await _ticketing.CreateTicket(
+                    new Dictionary<string, string>
+                    {
+                        { "Registry", "BuildingRegistry" },
```
Comment (emalfroy) [inline src/Grb.Building.Api/Uploads/UploadPreSignedUrlHandler.cs:61]: Change to basisregisters-grb

File: src/Grb.Building.Processor.Job/Dockerfile [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+
+# create group & user
+RUN addgroup --gid 1000 --system app && adduser --uid 1000 -system app --gid 1000
+
+# create work dir and set permissions as WORKDIR sets permissions as root
+RUN mkdir /app && chown -R app:app /app
+WORKDIR /app
+
+LABEL maintainer "Digitaal Vlaanderen <digitaal.vlaanderen@vlaanderen.be>"
+LABEL registry="building-registry"
```
Comment (emalfroy) [inline src/Grb.Building.Processor.Job/Dockerfile:11]: basisregisters-grb

File: CHANGELOG.md [hunk 8]
```diff
@@ -0,0 +1,3799 @@
...
+
+### Features
+
+* add grb jobrecord status ErrorResolved ([f4f0e62](https://github.com/informatievlaanderen/building-registry/commit/f4f0e6230ef650aeda50672e3d65ca14224cbeb9))
+* upload grb job result + archive grb job records ([ee1f705](https://github.com/informatievlaanderen/building-registry/commit/ee1f70505cbac5f4b164a36318b707d050673411))
+
+# [3.76.0](https://github.com/informatievlaanderen/building-registry/compare/v3.75.0...v3.76.0) (2023-05-17)
+
+
+### Bug Fixes
```
Comment (jvandaal) [inline CHANGELOG.md:17]: There is some stuff here which has no relevance to this repo.

File: src/EF.MigrationsHelper/EF.MigrationHelper.csproj [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+  <PropertyGroup>
+    <PackageId>EF</PackageId>
+    <Title>EF</Title>
+    <Authors>Basisregisters Vlaanderen</Authors>
+    <Description>Entity Framework Migrations Dummy Project.</Description>
+    <Copyright>Copyright (c) Vlaamse overheid</Copyright>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\BuildingRegistry.Projections.Wms\BuildingRegistry.Projections.Wms.csproj" />
```
Comment (jvandaal) [inline src/EF.MigrationsHelper/EF.MigrationHelper.csproj:28]: Cleanup needed

File: src/Grb.Building.Processor.Upload/Dockerfile [hunk 10]
```diff
@@ -0,0 +1,23 @@
...
+
+# create group & user
+RUN addgroup --gid 1000 --system app && adduser --uid 1000 -system app --gid 1000
+
+# create work dir and set permissions as WORKDIR sets permissions as root
+RUN mkdir /app && chown -R app:app /app
+WORKDIR /app
+
+LABEL maintainer "Digitaal Vlaanderen <digitaal.vlaanderen@vlaanderen.be>"
+LABEL registry="building-registry"
```
Comment (emalfroy) [inline src/Grb.Building.Processor.Upload/Dockerfile:11]: basisregisters-grb

File: src/EF.MigrationsHelper/appsettings.json [hunk 11]
```diff
@@ -0,0 +1,42 @@
+{
+  "ConnectionStrings": {
```
Comment (jvandaal) [inline src/EF.MigrationsHelper/appsettings.json:2]: Cleanup needed

File: src/Grb.Building.Api/Infrastructure/Startup.cs (hunk 1 above)
Comment (jvandaal) [reply src/Grb.Building.Api/Infrastructure/Startup.cs:79]: Don't think so. Parcel will probably also have its API.

File: src/Grb.Building.Processor.Upload/paket.references [hunk 12]
```diff
@@ -0,0 +1,37 @@
+AWSSDK.S3
+AWSSDK.Extensions.NETCore.Setup
+AWSSDK.ECS
+
+Destructurama.JsonNet
+
```
Comment (emalfroy) [inline src/Grb.Building.Processor.Upload/paket.references:6]: whitespace

**Review by pgallik — APPROVED** [review]

File: src/Grb.Building.Api/appsettings.json (hunk 2 above)
Comment (ArneD) [reply src/Grb.Building.Api/appsettings.json:28]: no, it's all good

File: src/Grb.Building.Api/Infrastructure/AmazonS3ExtendedClient.cs [hunk 13]
```diff
@@ -0,0 +1,205 @@
...
+
+            var credentials = Credentials.GetCredentials();
+
+            if (credentials is null)
+            {
+                _logger.LogError("No credentials found");
+            }
+
+            _logger.LogWarning($"credentials: {credentials.AccessKey}");
+            _logger.LogWarning($"usetoken: {credentials.UseToken}");
```
Comment (ArneD) [inline src/Grb.Building.Api/Infrastructure/AmazonS3ExtendedClient.cs:55]: Should be in the `else { }`

File: src/Grb.Building.Api/Infrastructure/AmazonS3ExtendedClient.cs [hunk 14]
```diff
@@ -0,0 +1,205 @@
...
+            : base(config)
+        {
+            _logger = loggerFactory.CreateLogger<AmazonS3ExtendedClient>();
+        }
+
+        public CreatePresignedPostResponse CreatePresignedPost(CreatePresignedPostRequest request)
+        {
+            var regionName = Config.RegionEndpoint.SystemName;
+
+            _logger.LogWarning($"region: {regionName}");
```
Comment (ArneD) [inline src/Grb.Building.Api/Infrastructure/AmazonS3ExtendedClient.cs:36]: Can you replace all warnings with information?

**Pull request merged**